src/pages/workspaces/WorkspacesList/useWorkspacesWithSubmissionStats.ts

```diff
@@ -0,0 +1,80 @@
+import { isAfter, parseJSON } from 'date-fns/fp';
+import _ from 'lodash/fp';
+import { useEffect, useMemo, useState } from 'react';
+import { useWorkspaces } from 'src/components/workspace-utils';
+import { Ajax } from 'src/libs/ajax';
+import { withErrorReporting } from 'src/libs/error';
+import { useCancellation } from 'src/libs/react-utils';
+import * as Utils from 'src/libs/utils';
```

Comment:
Might be worth switching now to https://github.com/DataBiosphere/terra-ui/blob/dev/packages/core-utils/src/logic-utils.ts#L20 for new TS code.

Reply:
src/libs/utils re-exports those functions from core-utils, so this should be getting the types.

But yeah, I'd suggest importing them directly from core-utils for new code. At some point, I'll have to clean up the re-export.